src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -243,15 +284,108 @@ public void banUser(String userName) {
+  public void clickOpenUserMessageWall() {
+    userMessageWallButton.click();
+    PageObjectLogging.log("clickOpenUserMessageWall", " Message wall button is clicked", true);
+  }
+
+  public void clickOpenUserContributions() {
+    userContributionsButton.click();
+    PageObjectLogging.log("clickOpenUserContributions", " Cotributions button is clicked", true);
+  }
+
+  public void clickOnUserOptionsKickButton() {
+    kickUserButton.click();
+    PageObjectLogging.log("clickOnUserOptionsKikButton", " kick user button is clicked", true);
+  }
+
+  public void switchToSecondTab(String firstTab){
+    ArrayList<String> openTabs = new ArrayList(driver.getWindowHandles());
+    for (String window:openTabs) {
+      if (window != firstTab) {
+        driver.switchTo().window(window);
+      }
+    }
+
+  }
+
+  public boolean isMessageWallOpened(String userName) {
+    try {
+      Assertion.assertEquals(userName, USER_NAME_ON_USER_PROFILE_SCREEN);
+      return userPageMessageWallTab.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageHederDisplayed() {
+    try {
+      return privateMessagesHeader.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isContributionsPageOpened (String userName) {
+    try {
+      Assertion.assertEquals(userName, USER_NAME_ON_USER_PROFILE_SCREEN);
+      return userPageContributionsTab.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isUserKickedFromChat() {
+    try {
+      return userBlockedMessageField.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isBlockedUserAbleToWritePrivateMesage() {
+    try {
+      return privateMassageButton.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isBlockPrivateMessageButtonDisplayed() {
+    try {
+      return blockPrivateMassageButton.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public void verifyStaffOptionsNotDisplayed(String userName){
+    clickOnDifferentUser(userName);
+    waitForElementNotVisibleByElement(userModOptions);
+    PageObjectLogging.log("verifyStaffOptionsNotDisplayed", userName + " mod options not displayed", true);
+  }
+
+  public boolean verifyUserCanNotBlockPrivateMessagesFromStaff(String userName){
+
+
+    waitForElementNotVisibleByElement(blockPrivateMassageButton);
+    PageObjectLogging.log("verifyUserCanNotBlockPrivateMessagesFromStaff",
+            userName + " can not block private messages from staff", true);
+  }
+
+  public boolean isChatUnbanMessageDispalyed() {
+    WebElement unBanMessage = (WebElement) By.xpath(String.format(USER_UNBAN_CONFIRM_MESSAGE, userName));
+    try {
+      return unBanMessage.isDisplayed();
+    }catch (ElementNotFoundException e){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)